package.json

```diff
@@ -18,16 +18,20 @@
   "types": "./dist/index.d.ts",
   "scripts": {
     "prepare": "husky install",
-    "postinstall": "if [ -e src/generator ]; then cd src/generator && npm install; fi",
     "test": "npm run generate && npm run test_without_generate",
     "test_without_generate": "jest src/parsing",
-    "generate": "cd src/generator && node main.js generic postgresql clickhouse",
+    "build_generator": "cd src/generator && rimraf dist && tsc -p tsconfig.build.json",
```

Comment:
1. We need multiple configs, because our main tsconfig.json checks ALL files, however in our build we want to transpile only certain files (for parsing and generators respectively).
2. We need to build it to make it a simple js file to run properly with node